Gemfile.lock

```diff
@@ -1,28 +1,29 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    activesupport (5.2.4.3)
+    activesupport (6.0.3.2)
```

Comment:
Do all tests pass? metadata interface is still at 5.x of the Rails-related gems, but at the moment I can't remember specifically why. It may have been due to issues with Sprockets.

Reply:
Yes, the tests (unit and acceptance) do all pass.  To be sure, I'll build this on integration and give it some manual testing as well.

That said, looking at the code, I'm not even sure if this particular project even uses ActiveRecord or Rails stuff at all.  It seems to be using Sinatra, so the activesupport bit may be unnecessary altogether.

Reply:
Just looked at the Gemfile.  We aren't even manually listing activesupport in the dependencies; it looks like it's being added as a depedency of turnip_formatter for the tests.  It doesn't appear to be used in anything else, from what I can see.  So I think we're good on this one.